desktop/apps/consignments2/client/actions.js

```diff
@@ -2,15 +2,31 @@ import { push } from 'react-router-redux'
 
 // Action types
 export const INCREMENT_STEP = 'INCREMENT_STEP'
+export const UPDATE_INPUTS = 'UPDATE_INPUTS'
 export const UPDATE_SUBMISSION = 'UPDATE_SUBMISSION'
 
 // Action creators
+export function createSubmission () {
+  return (dispatch) => {
+    // TODO: async request to convection to create submission
+    dispatch(updateSubmission()) // update state
+    dispatch(incrementStep()) // move to next step
+  }
+}
+
 export function incrementStep () {
   return {
     type: INCREMENT_STEP
   }
 }
 
+export function submitDescribeWork (values) {
+  return (dispatch) => {
+    dispatch(updateInputs(values)) // update the inputs in case we have to return
```

Comment:
Yeah, these map to the fields that we actually send to convection to create a submission, which may or may not be the same as what's on the form and also aggregate other parts of the submission flow (i.e. artist autocomplete/photo upload) which are separate UIs that probably won't need these form helpers.